docfiles/apptracking.html

```diff
@@ -29,14 +30,29 @@
             }
         });
 
-        var scriptIdRegex = /(?:S?\d{5}-\d{5}-\d{5}-\d{5})|(?:_[0-9a-zA-Z]{12})/g;
```

Comment:
Can we just bump this up to the beginning of `loadAppInsights` right next to `isProduction` instead of storing it on window? Typically better to avoid polluting global namespace when possible